src/Admin/Vue/PaymentMethods.php

```diff
@@ -15,7 +15,7 @@ public function payment_method_standard( ) {
 
 
 		$method = [
-			"redirect" => false,
```

Comment:
Nuno please read the code above!!! the `switch`!!!! You shouldn't be doing this here

Reply:
ok